README.md

```diff
@@ -229,6 +229,9 @@ You can also select the `Reattach children` checkbox to move along all children
 <summary><strong>Override fork point</strong></summary>
 <br/>
 
+Branch fork point is the commit from which given branch was created.
```

Comment:
Whoops... no, that's not always true :/ What if a long-existing branch has been rebased onto a parent branch that has itself been rebased a few times?...